pkg/ui/workspaces/cluster-ui/src/tableDetailsV2/tableOverview.tsx

```diff
@@ -0,0 +1,135 @@
+  } = useNodeStatuses();
+
+  // getNodesByRegionDisplayStr returns a string that displays
+  // the regions and nodes that the table is replicated across.
+  const getNodesByRegionDisplayStr = (): string => {
+    if (nodesLoading || !tableDetails?.metadata) {
+      return "";
+    }
+    const nodesByRegion: Record<string, number[]> = {};
+    metadata.store_ids.forEach(storeID => {
+      const nodeID = storeIDToNodeID[storeID as StoreID];
+      const region = nodeIDToRegion[nodeID];
+      if (!nodesByRegion[region]) {
+        nodesByRegion[region] = [];
+      }
+      nodesByRegion[region].push(nodeID);
+    });
+    return Object.entries(nodesByRegion)
+      .map(
+        ([region, nodes]) =>
+          `${region} (${nodes.map(nid => "n" + nid).join(",")})`,
+      )
+      .join(", ");
+  };
+
+  const percentLiveDataWithPrecision = (
+    metadata.percent_live_data * 100
+  ).toFixed(2);
+
+  const timezone = useContext(TimezoneContext);
+  const lastUpdatedText = FormatWithTimezone(
+    moment.utc(metadata.last_updated),
+    DATE_WITH_SECONDS_FORMAT_24_TZ,
+    timezone,
+  );
+  const formattedErrorText = metadata.last_update_error
+    ? "Update error: " + metadata.last_update_error
+    : "";
+
+  return (
+    <>
+      <PageSection>
+        <SqlBox
+          value={tableDetails.create_statement}
+          size={SqlBoxSize.CUSTOM}
+        />
+      </PageSection>
+      <PageSection>
+        <Row justify={"end"}>
+          <Col>
+            <Tooltip title={formattedErrorText}>
+              <Row gutter={8} align={"middle"} justify={"center"}>
+                {metadata.last_update_error && (
+                  <Icon fill="warning" iconName={"Caution"} />
+                )}
+                <Col>Last updated: {lastUpdatedText}</Col>
+              </Row>
+            </Tooltip>
+          </Col>
+        </Row>
+        <Row gutter={8}>
+          <Col span={12}>
+            <SummaryCard>
+              <SummaryCardItem
+                label="Size"
+                value={Bytes(metadata.replication_size_bytes)}
+              />
+              <SummaryCardItem label="Ranges" value={metadata.range_count} />
+              <SummaryCardItem
+                label="Regions / Nodes"
+                value={
+                  <Skeleton loading={nodesLoading}>
+                    {getNodesByRegionDisplayStr()}
+                  </Skeleton>
+                }
+              />
+            </SummaryCard>
+          </Col>
+          <Col span={12}>
+            <SummaryCard>
+              <SummaryCardItem
+                label="% of Live data"
+                value={
+                  <div>
+                    <div>{percentLiveDataWithPrecision}% </div>
+                    <div>
+                      {Bytes(metadata.total_live_data_bytes)} /{" "}
+                      {Bytes(metadata.total_live_data_bytes)}
+                    </div>
+                  </div>
+                }
+              />
```

Comment:
Is auto stats collections / stats last updated coming in a future PR?

Reply:
Yup! This PR was originally udpated before you merged your changes I believe - I'll update in a follow-up so we don't have to rerun CI as to not block this and the grants page.